src/main/python/vantiqsdk.py

```diff
@@ -1044,6 +1047,7 @@ async def count(self, resource: str, where: Union[dict, None]) -> VantiqResponse
             method = 'GET'
             path = self._build_path(resource, None)
             resp = await self._perform_operation(operation, method, path, query_params, False)
+            resp.body = None  # After a count, we don't need to return the data fetched (since we decided it anyway)
```

Comment:
Will this appear as an empty dictionary in the response object? I noticed that the delete call returns an empty object for the response body, I was wondering this was the same.
![image](https://user-images.githubusercontent.com/51422510/170373026-b1760700-65e4-49ad-9b2c-2712aee8d7f4.png)

Reply:
This just returns `None` rather than an empty dictionary object.  But it may makes sense to make it analogous to the `delete` function, so I'll tweak appropriately.